examples/verify_addresses_from_csv/verify.py

```diff
@@ -0,0 +1,70 @@
+# Usage
+# python verify.py input.csv
+
+import csv
+import lob
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(__file__+'../../../..'))
+
+lob.api_key = 'test_fc26575412e92e22a926bc96c857f375f8b'
+
+skipFirstLine = True
+
+# Column indices in CSV
+address1 = 0
+address2 = 1
+city = 2
+state = 3
+postcode = 4
+country = 'US'
+
+try:
+    sys.argv[1]
+except IndexError:
+    print("Please provide an input CSV file as an argument.")
+    print("Usage: python verify.py <csv-input>")
+    sys.exit(1)
+
+# Open input files
+inputFile = open(sys.argv[1], 'rU')
+csvInput = csv.reader(inputFile)
+
+# Create output files
+cwd = os.path.dirname(os.path.abspath(__file__))
+errors = open(cwd + '/errors.csv', 'w')
```

Comment:
Can we use the same structure for output files? I like that it generates a new folder for each iteration in the format of `output/<timestamp>/errors.csv`